packages/zapp/console/src/components/Entities/EntityVersionDetails.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react';
+import { withRouteParams } from 'components/common/withRouteParams';
+import { ResourceIdentifier, ResourceType } from 'models/Common/types';
+import { makeStyles, Theme } from '@material-ui/core/styles';
+import { WaitForData } from 'components/common/WaitForData';
+import { useProject } from 'components/hooks/useProjects';
+import { StaticGraphContainer } from 'components/Workflow/StaticGraphContainer';
+import { WorkflowId } from 'models/Workflow/types';
+import { entitySections } from 'components/Entities/constants';
+import { EntityDetailsHeader } from 'components/Entities/EntityDetailsHeader';
+import { EntityVersions } from 'components/Entities/EntityVersions';
+import { typeNameToEntityResource } from './constants';
+
+const useStyles = makeStyles((_theme: Theme) => ({
```

Comment:
nit: switch `_theme` with `theme` as it's used on line 20

Reply:
tks! solved